lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -69,9 +69,9 @@ public class ProcessJourneyEventHandler
     public static final String CURRENT_PAGE = "currentPage";
     private static final String CORE_SESSION_TIMEOUT_STATE = "CORE_SESSION_TIMEOUT";
     private static final String NEXT_EVENT = "next";
-    private static final String END_SESSION_EVENT = "build-client-oauth-response";
-    private static final StepResponse END_SESSION_RESPONSE =
-            new ProcessStepResponse(END_SESSION_EVENT, null);
+    private static final String BUILD_CLIENT_OAUTH_RESPONSE_EVENT = "build-client-oauth-response";
+    private static final StepResponse BUILD_CLIENT_OAUTH_RESPONSE =
+            new ProcessStepResponse(BUILD_CLIENT_OAUTH_RESPONSE_EVENT, null);
```

Comment:
I see someone has 'fixed' this - I deliberately made this a separate constant, because the one above is for the journey event, and this one is for the journey response type (they just happen to be the same!)

Fine to leave..!

Reply:
Not sure I follow?

Reply:
There are two separate constants here:
- The journey event that triggers this special behaviour (`build-client-oauth-response`)
- The response emitted - a process step with `lambda: build-client-oauth-response`

The two strings match, but they are not conceptually 'the same' - e.g. if we rename the lambda, one will change but the other won't. Just me being grumpy and pedantic :)

Reply:
If we changed the journey event to something explicit like `recovery-return-to-rp` then it'd be a lot less confusing

Reply:
Yeah agreed. I think lets get this out and then look at fixing it up - will need co-ordination with the frontend